lib/subscriptions_test_kit/suites/subscriptions_r5_backport_r4_server/subscription_rejection_group.rb

```diff
@@ -15,7 +21,14 @@ class SubscriptionRejectionGroup < Inferno::TestGroup
                   :unsupported_subscription_channel_endpoint, :unsupported_subscription_payload_type,
                   :unsupported_subscription_channel_payload_combo
 
-      test from: :subscriptions_r4_server_reject_subscriptions
+      # test from: :subscriptions_r4_server_reject_subscriptions
```

Comment:
remove comment

Reply:
Addressed in [c05d5d6](https://github.com/inferno-framework/subscriptions-test-kit/pull/7/commits/c05d5d6903172f295c2cacb861d85b939b802630)